scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -111,17 +112,17 @@
 
 # In[9]:
 
-get_ipython().run_cell_magic('time', '', "path = os.path.join('data', 'expression-matrix.tsv.bz2')\nX = pd.read_table(path, index_col=0)")
+get_ipython().run_cell_magic('time', '', "path = os.path.join('download', 'expression-matrix.tsv.bz2')\nX = pd.read_table(path, index_col=0)")
```

Comment:
Shouldn't be as the upstream changes to `3.TCGA-MLexample_Pathway.ipynb` are part of this pull request.